### DISCUSSION TITLE
Welcome to usher Discussions!

### DISCUSSION THREAD

**Discussion by yatisht** [yatisht/usher#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dezordi** [comment]
Hey all,

My name is Filipe and I'm a PhD student working with bioinformatics applied to mosquito and viral genomics.

Today I'm doing some tests with usher (congratulations to the team, it's was a beautiful tool and I was able to easily implement with docker in our local server ). 

I'm wondering how usher deals with genomes that have insertions, considering that for unaligned sequences, the python script Fasta2UShER.py will perform a mafft alignment with the --keeplength parameter.

So, considering the --keep length parameter, I think that the reference global phylogeny was created with an alignment generated following the length of SARS-CoV-2  NC_045512.2 reference genome right? And if in a near future we had some SARS-CoV-2 lineages characterized by insertions, how usher will work with this kind of information?

Best regards,

Filipe
